lib/src/http/managers/emoji_manager.dart

```diff
@@ -89,13 +90,13 @@ class EmojiManager extends Manager<Emoji> {
   }
 
   @override
-  Future<GuildEmoji> create(EmojiBuilder builder, {String? audiReason}) async {
+  Future<GuildEmoji> create(EmojiBuilder builder, {String? auditLogReason}) async {
```

Comment:
This is a breaking change

Reply:
these parameters was previously ignored and was incorrectly spelled

Reply:
I believe it's OK if we let this through -- nobody would have been using these parameters as they were both misspelled and did nothing.